vcxagency-base/vcxagency-base.Dockerfile

```diff
@@ -1,90 +1,73 @@
-FROM ubuntu:16.04 as BASE
-
-ARG uid=1000
-
-RUN apt-get update && \
-    apt-get install -y \
-      pkg-config \
-      libssl-dev \
-      libgmp3-dev \
-      curl \
-      build-essential \
-      libsqlite3-dev \
-      cmake \
-      git \
-      python3.5 \
-      python3-pip \
-      python-setuptools \
-      apt-transport-https \
-      ca-certificates \
-      debhelper \
-      wget \
-      devscripts \
-      libncursesw5-dev \
-      libzmq3-dev \
-      zip \
-      unzip \
-      jq
-
-
-RUN pip3 install -U \
-	pip \
-	setuptools \
-	virtualenv \
-	twine \
-	plumbum \
-	deb-pkg-tools
-
-RUN cd /tmp && \
-   curl https://download.libsodium.org/libsodium/releases/libsodium-1.0.18.tar.gz | tar -xz && \
-    cd /tmp/libsodium-1.0.18 && \
-    ./configure --disable-shared && \
-    make && \
-    make install && \
-    rm -rf /tmp/libsodium-1.0.18
-
-RUN useradd -ms /bin/bash -u $uid indy
-USER indy
+FROM alpine:3.12 AS builder
+
+ARG UID=1001
+ARG GID=1001
+
+ARG INDYSDK_PATH=/home/indy/indy-sdk
+ARG INDYSDK_REPO=https://github.com/hyperledger/indy-sdk.git
+ARG INDYSDK_REVISION=v1.15.0
+
+ENV LC_ALL="C.UTF-8"
+ENV LANG="C.UTF-8"
+ENV RUST_LOG=warning
 
-RUN curl https://sh.rustup.rs -sSf | sh -s -- -y --default-toolchain 1.43.1
-ENV PATH /home/indy/.cargo/bin:$PATH
+RUN addgroup -g $GID indy && adduser -u $UID -D -G indy indy
 
-ARG INDYSDK_REVISION
-ARG INDYSDK_REPO
+RUN apk update && apk upgrade && \
+    apk add --no-cache \
+        build-base \
+        cargo \
+        git \
+        libsodium-dev \
+        libzmq \
+        openssl-dev \
+        rust \
+        sqlite-dev \
+        zeromq-dev
+
+USER indy
 WORKDIR /home/indy
-RUN git clone "${INDYSDK_REPO}" "./indy-sdk"
-RUN cd "/home/indy/indy-sdk" && git checkout "${INDYSDK_REVISION}"
 
-RUN cargo build --release --manifest-path=/home/indy/indy-sdk/libindy/Cargo.toml
+RUN git clone $INDYSDK_REPO && \
+    cd indy-sdk && git checkout $INDYSDK_REVISION
+
+RUN cargo build --release --manifest-path=$INDYSDK_PATH/libindy/Cargo.toml
+
 USER root
-RUN mv /home/indy/indy-sdk/libindy/target/release/*.so /usr/lib
+RUN mv $INDYSDK_PATH/libindy/target/release/libindy.so /usr/lib
+
 USER indy
-RUN cargo build --release --manifest-path=/home/indy/indy-sdk/experimental/plugins/postgres_storage/Cargo.toml
-USER root
-RUN mv /home/indy/indy-sdk/experimental/plugins/postgres_storage/target/release/*.so /usr/lib
+RUN cargo build --release --manifest-path=$INDYSDK_PATH/experimental/plugins/postgres_storage/Cargo.toml
 
-FROM ubuntu:16.04
+RUN mv $INDYSDK_PATH/experimental/plugins/postgres_storage/target/release/libindystrgpostgres.so .
```

Comment:
Can we rather for sake of clarity keep `mv` of this lib to `/usr/lib` to be consistent with location of `libindy`?

Reply:
Uglified as requested: https://github.com/marekprague/vcxagencynode/commit/a198e6b0150d426d3cc0c551df0077ae7983f303

@Patrik-Stas can you merge now?